tests/apm_tracing_e2e/test_otel.py

```diff
@@ -3,8 +3,8 @@
 
 
 @missing_feature(
-    context.weblog_variant not in "net-http",
-    reason="The /e2e_otel_span endpoint is only implemented in Go net/http at the moment.",
+    context.weblog_variant not in ("net-http", "spring-boot"),
```

Comment:
You might and to use `context.weblog_variant.startswith("spring-boot")` if you want to get this for all variants we have using the Spring Boot app (jetty,  undertow, payara, etc).

Reply:
This test is using OpenTelemetry API and don't rely on / ignore the other auto instrumentation generated spans.

So does it brings something to add the other variants too?
(note that would required to enable OTEL from their Dockerfile too I guess?)

Reply:
I don’t think we need to test this on multiple variants. As you said, it’s only testing the OTel API.